lib/internal/http2/util.js

```diff
@@ -254,43 +254,43 @@ function getDefaultSettings() {
   const flags = settingsBuffer[IDX_SETTINGS_FLAGS];
 
   if ((flags & (1 << IDX_SETTINGS_HEADER_TABLE_SIZE)) ===
-      (1 << IDX_SETTINGS_HEADER_TABLE_SIZE)) {
+    (1 << IDX_SETTINGS_HEADER_TABLE_SIZE)) {
```

Comment:
Unrelated whitespace change here...
```suggestion
      (1 << IDX_SETTINGS_HEADER_TABLE_SIZE)) {
```